src/services/__tests__/map.service.test.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  mapPlayers, mapTerms, mapScorepad, mapScorepads,
+} from '../map.service';
+import apiPlayers from '../../../mocks/apiPlayers';
+import players from '../../../mocks/players';
+import apiTerms from '../../../mocks/apiTerms';
+import terms from '../../../mocks/terms';
+import apiScorepad from '../../../mocks/apiScorepad';
+import scorepad from '../../../mocks/scorepad';
+import apiScorepads from '../../../mocks/apiScorepads';
+import scorepads from '../../../mocks/scorepads';
+
+describe('players mapper', () => {
+  it('returns a mapped player object', async () => {
+    const expectedResult = players;
+    const unmappedPlayers = apiPlayers;
+    const result = mapPlayers(unmappedPlayers);
+
+    expect(result).toStrictEqual(expectedResult);
```

Comment:
I added tests for empty arrays. I could not find other test cases because the map functions enforce the input, so I could not try to force an error by using a bad input